integration_tests/tests/forseti_integration_testing/controls/inventory.rb

```diff
@@ -0,0 +1,493 @@
+require 'json'
+control 'inventory' do
+
+     describe "Create an inventory" do
```

Comment:
Can this this test should be combined with `Create an inventory and check database` below?  Can we have multiple `do` commands and assert on them within one test?  Otherwise, the tests here are very repetitive.

For example why have a separate purge test where inventory is created again, if we can test the purge command here?

Reply:
Tests have been rewritten to create inventory / model once at the beginning and make assertions on that to test inventory and model Forseti commands.